.gitignore

```diff
@@ -79,3 +79,5 @@ bin/*
 
 # External downloaded libs (TPL downloaded libs, etc...)
 external_libraries/tetgen/*
+
+applications/ThermalDEMApplication/DEM_HeatTransferModels.pdf
```

Comment:
instead of modifying this file in the root folder, you can ignore this file locally by editing `.git/info/exclude`.

Reply:
In the end, I decided not to ignore the pdf. I didn't know I made this modification in this file, now I removed.